src/WrapperApp/components/SimulationPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Button, LinearProgress } from '@mui/material';
+import ky from 'ky';
+
+import React, { useState } from "react";
+import { useStore } from '../../services/StoreService';
+import { BACKEND_URL } from '../../util/Config';
+
+interface SimulationPanelProps {
+    onError?: (error: unknown) => void;
+    onSuccess?: (result: unknown) => void;
+}
+
+export default function SimulationPanel(props: SimulationPanelProps) {
+    const { editorRef } = useStore();
+    const [isInProgress, setInProgress] = useState<boolean>(false);
+
+    const sendRequest = () => {
+        setInProgress(true);
+        ky.post(`${BACKEND_URL}/sh/demo`, { json: editorRef.current?.toJSON() })
+            .json()
+            .then((response) => {
+                console.log(response);
```

Comment:
Can't we use some modal popup for debugging purposes here ?

Reply:
Yes we can.